app/assets/javascripts/refresh_token.js.coffee

```diff
@@ -14,9 +14,9 @@
 
 $ ->
   refreshToken = ->
-    token = currentPlayer.media.src.split('?')[1]
+    token = currentPlayer.node.currentSrc.split('?')[1]
     if token && token.match(/^token=/)
-      mount_point = $('body').data('mountpoint')
+      mount_point = $('body').data('mountpoint') || '/'
```

Comment:
Just sanity checking, but if some devious person manipulates this script, or the mountpoint in the HTML, to request arbitrary files, there's something on the server end to keep it only returning these authorize.txt files, yes?

Mostly picturing something like someone issuing a `$.get('/etc/password')` kind of thing

Reply:
Yeah, it routes the thing for the `authorize.txt` below via:

https://github.com/ualbertalib/avalon/blob/avalon-6/config/routes.rb#L52
https://github.com/ualbertalib/avalon/blob/avalon-6/app/controllers/derivatives_controller.rb#L24

No actual text file (or any other file) is fetched from the host.